charts/ingresslibchart/templates/_ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- define "libchart.ingress.tpl" -}}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Is there a nonbeta version we can use? I've gotten some warnings here and there about beta ingress being deprecated and unavailable in future k8s versions.